addOns/client/src/main/java/org/zaproxy/addon/client/ClientIntegrationAPI.java

```diff
@@ -138,6 +139,17 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
                 this.extension.addReportedObject(new ReportedEvent(json));
                 break;
 
+            case ACTION_REPORT_ZEST_SCRIPT:
+                String scriptJson = this.getParam(params, PARAM_SCRIPT_JSON, "");
+                LOGGER.debug("Got script: {}", scriptJson);
+                json = JSONObject.fromObject(scriptJson);
```

Comment:
This should be done inside the try-catch block but we should not need to parse at this stage, let Zest do that.